pkg/virtctl/portforward/portforwarder.go

```diff
@@ -0,0 +1,93 @@
+package portforward
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"strings"
+
+	"kubevirt.io/client-go/kubecli"
+)
+
+type portForwarder struct {
+	namespace, name string
+	resource        portforwardableResource
+}
+
+type portforwardableResource interface {
+	PortForward(name string, port int) (kubecli.StreamInterface, error)
+}
+
+func (p *portForwarder) startForwarding(address *net.IPAddr, port forwardedPort) error {
+	if port.protocol == protocolUDP {
+		// TODO: implement udp
```

Comment:
kubectl portforward does not support UDP right now.
I think we can support it if we want and so far I've kept the option open, just didn't do it yet.
Any thoughts if we should support it? Would it be needed for specific use cases like RDP?